styles/portfolio.css

```diff
@@ -0,0 +1,93 @@
+body {
+    margin: 0;
+    font-family: helvetica;
+    display: grid;
+    grid-template-columns: 20% auto 20%;
+    grid-template-rows: .5fr 1fr .25fr;
+    color: #3e3e3e;
+    font-weight: 300;
+}
+
+header {
+    text-align: center;
+    margin-top: 5em;
+    grid-column: 2/3;
+    grid-row: 1/2;
+}
+
+header img {
+    width: 100vw;
+}
+
+header nav li {
+    background-color: #E2E2F3;
+    padding: .5em;
+    border-radius: 10px;
+}
+
+ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+li {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+    color: #3e3e3e;
+}
+
+main {
+    grid-column: 1/4;
+    grid-row: 2/3;
+    background-color: #E2E2F3;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+section {
+    width: 500px;
+}
+
+section img {
+    width: 100%;
+}
+
+footer {
+    grid-column: 1/4;
+    grid-row: 3/ span 4;
+    background-color: #3e3e3e;
+}
+
+footer a {
+    color: whitesmoke;
+}
+
+.tags li {
+    color: whitesmoke;
+    border-radius: 10px;
+    padding: .5em;
+    font-size: .7em;
+    font-weight: bold;
+}
+
+.tags li:nth-of-type(1) {
+    background-color: royalblue;
+}
+
+.tags li:nth-of-type(2) {
+    background-color: orange;
+}
+
+.tags li:nth-of-type(3) {
+    background-color: mediumvioletred;
+}
+
+.tags li:nth-of-type(4) {
+    background-color: teal;
+}
```

Comment:
Consider moving the similar tag stuff from about into a shared css file that can be linked to from both the about and portfolio pages.